src/components/PeriodicReports/ReportLabel.tsx

```diff
@@ -7,11 +9,21 @@ import {
   SecuredPeriodicReportFragment,
 } from './PeriodicReport.generated';
 
+const useStyles = makeStyles(() => ({
```

Comment:
We use this component in table rows, in cards, and in titles. What do you think? Does this extra styling make sense in all those places?

Reply:
reverted and implemented rendering in `PeriodicReportsTable`.